scripts/shallow-diff-base-smoke.sh

```
@@ -141,7 +141,14 @@ set +e
 full_code=$?
 set -e
 echo "full_exit=${full_code}" >>"${receipt}"
-[[ "${full_code}" -eq 0 ]] || fail "full-history diff failed (exit ${full_code}); see ${full_log}"
+if [[ "${full_code}" -ne 0 ]]; then
+  echo "full_history_positive=fail" >>"${receipt}"
```

Shellcheck (0.11.0)

[style] Consider using { cmd1; cmd2; } >> file instead of individual redirects.

(SC2129)